docs/querying/projections.md

```diff
@@ -0,0 +1,179 @@
+
+You can define a projection after you ingest data. Although you can define the projection in a compaction spec, we recommend using the [`druid-catalog`](../development/extensions-core/catalog.md) extension.
+
+The following API call includes a payload with the `properties.projections` block that defines your projections:
+
+<details>
+<summary>View the payload</summary>
+
+```json {11,19,39} showLineNumbers
+{
+  "type": "datasource",
+  "columns": [],
+  "properties": {
+    "segmentGranularity": "PT1H",
+    "projections": [
+      {
+        "spec": {
+          "name": "channel_page_hourly_distinct_user_added_deleted",
+          "type": "aggregate",
+          "virtualColumns": [
+            {
+              "type": "expression",
+              "name": "__gran",
+              "expression": "timestamp_floor(__time, 'PT1H')",
+              "outputType": "LONG"
+            }
+          ],
+          "groupingColumns": [
+            {
+              "type": "long",
+              "name": "__gran",
+              "multiValueHandling": "SORTED_ARRAY",
+              "createBitmapIndex": false
+            },
+            {
+              "type": "string",
+              "name": "channel",
+              "multiValueHandling": "SORTED_ARRAY",
+              "createBitmapIndex": true
+            },
+            {
+              "type": "string",
+              "name": "page",
+              "multiValueHandling": "SORTED_ARRAY",
+              "createBitmapIndex": true
+            }
+          ],
+          "aggregators": [
+            {
+              "type": "HLLSketchBuild",
+              "name": "distinct_users",
+              "fieldName": "user",
+              "lgK": 12,
+              "tgtHllType": "HLL_4"
+            },
+            {
+              "type": "longSum",
+              "name": "sum_added",
+              "fieldName": "added"
+            },
+            {
+              "type": "longSum",
+              "name": "sum_deleted",
+              "fieldName": "deleted"
+            }
+          ]
+        }
+      }
+    ]
+  }
+}
+```
+
+</details>
+
+In this example, Druid aggregates data into `distinct_user`, `sum_added`, and `sum_deleted` dimensions based on the aggregator that's specified and a source dimension. These aggregations are grouped by the columns you define in `groupingColumns`.
+
+## Use a projection 
+
+Druid automatically uses a projection if your query matches a projection you've defined. There are some query context parameters that give you some control on how projections are used and Druid's behavior:
+
+- `useProjection`: The name of a projection you defined. The query engine must use that projection and will fail the query if the projection does not match the query.
+- `forceProjections` `true` or `false`. The query engine must use a projection and will fail the query if there isn't a matching projection.
+-  `noProjections`:  `true` or `false`. The query engine won't use any projections.
```

Comment:
```suggestion
- `noProjections`:  Set to `true` to prevent the query engine from using projections altogether. Defaults to `false`.
```
seriously this should be useProjections which defaults to true and you could set to false to disable. @clintropolis :/